src/components/map/MapView.js

```diff
@@ -27,35 +28,45 @@ const MapView = props => {
     const mapControls = getMapControls(isPlugin, isSplitView, controls);
 
     return (
-        <Fragment>
-            {isSplitView ? (
-                <SplitView
-                    isPlugin={isPlugin}
-                    isFullscreen={isFullscreen}
-                    basemap={basemap}
-                    layer={splitViewLayer}
-                    controls={mapControls}
-                    feature={feature}
-                    openContextMenu={openContextMenu}
-                    resizeCount={resizeCount}
-                />
+        <>
+            {basemap.id === undefined ? (
+                <ComponentCover translucent>
```

Comment:
Remove `translucent`. The spinner looks better without it when it's no content to cover.